### PR TITLE
New blog post: Let's Dev: A Package Manager

### PR DESCRIPTION
**Pull request by arcanis**

### CODE REVIEW

**Comment by bestander** [comment]
Woohoo!

**Comment by arcanis** [comment]
@bestander @Haroenv do you know what's the right way to trigger a PR preview on netlify? Is it automatic? Also, I've used the `announcement` category since I haven't found any other in previous posts, but maybe another one is more appropriate :)

**Comment by bestander** [comment]
As for netify, I think the build takes like 10-20 min

**Comment by arcanis** [comment]
Still not generated :<

**Comment by Daniel15** [comment]
Strangely I don't even see this PR in Netlify's queue 😕

**Comment by Haroenv** [comment]
I have no idea why it's not in the queue. Force pushing or something will probably trigger it

File: _posts/2017-07-11-lets-dev-a-package-manager.md [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+share_text : "Let's Dev: A Package Manager"
+---
+
+Hello everyone! Today, we're gonna write a new package manager, even better than Yarn! Ok, maybe not, but at least we're gonna have some fun, learn how package managers work, and think about what could come next on Yarn.
+
+> **The devil is in the details**
+>
+> This article omits small details and environment quirks, and focuses on the high-level architecture of a package manager, in an effort to stay succinct. For example, we're gonna assume that all paths are regular POSIX paths.
+> 
+> That being said, there's much to say about these compatibility layers, and maybe talking about them could be an interesting follow up! Feel free to tweet at @yarnpkg if you're interested to know more about them! 😃
```
Comment (Haroenv) [inline _posts/2017-07-11-lets-dev-a-package-manager.md]: Maybe make that @ a link

**Comment by Haroenv** [comment]
Deploy preview ready!

Built with commit 74dd5cbe57549e3abdeaacecb60f9d9ef558593b

https://deploy-preview-568--yarnpkg.netlify.com

**Comment by Haroenv** [comment]
the deploys are running again now @arcanis, I also set up "commenting" since, GitHub status doesn't work because I don't have admin access to this repo I think (cc @Daniel15). I didn't know that that would comment under my name though

**Comment by Haroenv** [comment]
Your name works now @arcanis

**Comment by arcanis** [comment]
Thanks @Haroenv ! That's super weird, it was displaying an empty string before, but worked when I tried with Konstantin ... maybe there was an invisible character ...

**Pull request merged**